app/models/campaign.rb

```diff
@@ -172,7 +172,8 @@ def days_left
     (end_datetime.to_date - Date.today).to_i
   end
 
-  def finished?
+  #The campaign has reached its end date time
+  def deadline_has_been_reached?
```

Comment:
I don't see an advantage to this change. It seems to be more wordy without much benefit. I think `finished?` makes more sense.